pkg/datapath/linux/config/config.go

```diff
@@ -901,10 +901,6 @@ func (h *HeaderfileWriter) writeTemplateConfig(fw *bufio.Writer, e datapath.Endp
 		fmt.Fprintf(fw, "#define ENABLE_ROUTING 1\n")
 	}
 
-	if !e.DisableSIPVerification() {
```

Comment:
With this, I think we can just remove this from the `EndpointConfiguration` interface.

Reply:
sounds good, let me try to fix that